README.md

```diff
@@ -5,368 +5,17 @@ https://www.simplereport.gov/
-
-
-## Load Tests
-SimpleReport leverages Locust for running its load tests. Currently, Locust starts as part of the local development build, and spawns three separate worker containers for inundating the system.
-
-To compose Locust with the rest of the development stack, run `yarn locust` instead of `yarn start`. *(NOTE: All other prerequisites listed above in "Developing locally" are still required for tests to function.)*
-
-To access the Locust user interface, navigate to `http://localhost:8089/`. Load tests will not kick off automatically; they must be manually invoked.
-
-Tests can be configured by modifying `backend/locust/locustfile.py`.
-
-
-## Twilio
-
-Twilio's Java SDK auto-configures based on two environment variables: `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN`. SMS is also disabled by default, and can be enabled in application.yml:
-
-```
-twilio:
-  enabled: true
-  from-number: +13214560987
-```
-
-These can also be set by environment variable if desired.
-
-## MailHog
-
-MailHog is an email-testing tool with a fake SMTP server underneath, we can use it to test sending emails locally.
-
-- Mailhog client runs on docker, `docker compose up -d mailhog`
-
-- Access mailhog inbox on `http://localhost:8025/`
-
-## Frontend
-
-The front end is a React app. The app uses [Apollo](https://www.apollographql.com/) to manage the graphql API. For styling the app leverages the [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/). Ensure your node version is version 14 in order for
-yarn to build packages correctly.
-
-### Linters
-
-This project uses [eslint](https://eslint.org/), [prettier](https://prettier.io/), and [stylelint](https://stylelint.io/) as frontend linters,
-and [spotless](https://github.com/diffplug/spotless) and [google-java-format](https://github.com/google/google-java-format) for the backend.
-GitHub Actions is configured to run these linters on every pull request, so you must resolve all mismatches/errors prior to merging.
-There are a few ways to manage this:
-
-1. Run `yarn lint:write` in the `frontend/` dir, and `./gradlew spotlessApply` in the `backend/` dir, before every commit
-1. Enable the optional pre-commit hook by running `yarn lefthook install` in the root dir (install lefthook if you haven't already done so during setup by running `yarn install`)
-1. Add extensions to your code editor that runs the linters for you on save, e.g. [prettier-vscode](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode), [vscode-eslint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint), [vscode-google-java-format](https://marketplace.visualstudio.com/items?itemName=ilkka.google-java-format)
-
-You can also run the pre-commit hooks in docker using `yarn lefthook run in-docker`.
-
-### Storybook and Chromatic
```

Comment:
⚠️ Moved to https://github.com/CDCgov/prime-simplereport/wiki/Storybook-and-Chromatic